.github/workflows/ci.yml

```
@@ -0,0 +1,231 @@
+      with:
+        python-version: "3.13"
+
+    - name: Install dependencies
+      run: uv pip install -e ".[dev]"
+
+    - name: Test CLI commands
+      run: |
+        # Test main CLI functionality
+        uv run python lightfast_mcp_manager.py init
+        uv run python lightfast_mcp_manager.py list
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)